keyboards/soldered/macro_pad/keyboard.json

```
@@ -0,0 +1,41 @@
+{
+    "manufacturer": "Soldered Electronics",
+    "keyboard_name": "Soldered Macro Pad",
+    "maintainer": "SolderedElectronics",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP18", "pin_b": "GP20", "resolution": 4}
```

Resolution isn't needed, as 4 is the default. 
```suggestion
            {"pin_a": "GP18", "pin_b": "GP20"}
```